src/CardArea/CardArea.css

```diff
@@ -1,4 +1,9 @@
 .card-area {
   background-color: lightgray;
   height: 100vh;
+}
+
+.card-area--section {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
I added this CSS rule so that the cards and decks would be displayed horizontally instead of vertically.